src/app/tests/suites/certification/Test_TC_ICDM_3_1.yaml

```diff
@@ -35,6 +35,47 @@ tests:
           2c.TH reads from the DUT the RegisteredClients attribute. Verify that the DUT response contains empty list of registered clients.
       disabled: true
 
+    - label:
+          "Step 0a: Preconfition 1: Read 'register-clients' by sending the
+          command below and note the check-in nodeID for the entries"
+      verification: |
+          ./chip-tool icdmanagement read registered-clients 1 0
```

Comment:
This seems like a brand new manual test. We should not add manual tests anymore

Reply:
Got it, though this script wasn't added in this PR.

Reply:
Manual step 0a is added by this PR even if the rest of the scripts were not part of this PR.

Not sure what the rules should be here ... I am defaulting to complaining about any manual verification step overall.

Reply:
@naveenkommareddi any opinions here? This adds a step part of a larger verification ... given 0a it seems like a prerequisite (however I did not read through the test plan) ... what do we do here? allow as one-off or stay strong on "no manual bits" and wait for this to be automated?

Reply:
tagging @cjandhyala as well

Reply:
> tagging @cjandhyala as well

@andy31415 this TC executed in 1.3 SVE and the script is updated based on SVE feedback. We will apply 'no manual TC bits' to the new 1.4 TCs.  Given it's already in 1.3 package we should let this change in so that ATLs will use the correct version of the script.

Reply:
Please confirm leaving it in or otherwise.